srcf/controllib/jobs.py

```diff
@@ -746,9 +746,9 @@ def new(cls, requesting_member, society, email):
             "email": email
         }
         require_approval = (
-            requesting_member.danger or
-            society.danger or
-            (email and email.endswith(("@cam.ac.uk", "@hermes.cam.ac.uk")))
+            requesting_member.danger
+            or society.danger
+            or (email and email.endswith(("@cam.ac.uk", "@hermes.cam.ac.uk")))
         )
```

Comment:
This is a backwards step in terms of readability.  Binary operators should go on the end of the previous line, which also leaves the terms tabulated.

Reply:
Any style which recommends/enforces binary operators at the start of lines can safely be ignored as broken.

Reply:
As it stands, both conventions (leading and trailing binary operators) are currently in use (for instance, [jobs.py](https://github.com/SRCF/srcf-python/blob/4070a88551df980fdf305a9df90f4a0d5f2135ca/srcf/controllib/jobs.py#L525) has a leading `or`). Would disabling just W504 or both W503 and W504 be the preferable choice?